### DISCUSSION TITLE
How to add Favorites to the sidebar?

### DISCUSSION THREAD

**Discussion by leethobbit** [jeffvli/feishin#424] [Q&A]
Hello - I am a Sonixd user and I frequently shuffle my Favorited tracks using the Favorites menu available there.  I have been testing out Feishin for the eventual switch but I can't seem to find an easy way to do this - is there a toggle somewhere I'm missing? Is it a planned feature? Thanks so much!

**Comment by basdp** [comment]
Add a smart playlist with this settings:
Mach all
Is Favorite - is - true

**Reply by kylian-002** [reply]
Thanks been looking for this for a while. I think this shut have been a default though
